lib/index.ts

```diff
@@ -281,6 +281,17 @@ function collectDiagnostics(program: ts.Program) {
   });
 }
 
+function checkUpdate(src: string, out: string): boolean {
```

Comment:
Small suggestion -- for clearer (more self-documenting) names, I suggest `needsUpdate` for the function, and `srcPath`, `outPath` for its parameters.